emva1288/process/data.py

```diff
@@ -275,24 +277,46 @@ def _get_spatial_data(self, d, postfix=''):
         are:
                - *'sum'*: the sum image preserved from input,
                - *'L'*: the number of image summed,
+               - *'M'*: the number of rows for one image
+               - *'N'*: the number of columns for one image
                - *'avg_var'*: variance of the the average image
                - *'avg_mean'*: mean of the the average image
                - *'var_mean'*: the variance of the variance image
+               - *'avg_var_cav'*: residual column spatial variance
+               - *'avg_var_rav'*: residual row spatial variance
         """
 
         # This cast is just in case the original images are unint
-        sum_ = d['sum'].astype(np.int64)
+        sum_ = d['sum'].astype(np.int64)  # <y> --> eqn(33)
```

Comment:
The comment is a bit misleading, since `avg_` is `<y>` from Equation 33. This is `sum(y[l])`.

Reply:
yes, that's the comment I marked that this value is eventually used to get eqn(33). I can remove it if you think it's confusing.

Reply:
Here's my suggestion.
```suggestion
        sum_ = d['sum'].astype(np.int64)  # y[l] sum from eqn(33)
```